Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -3501,6 +3501,20 @@ InvisibleAutoplayNotPermitted:
     WebCore:
       default: false
 
+InvokeAttributeEnabled:
```

What do you think of this name?

```suggestion
InvokerAttributesEnabled:
```